aws-lambda-layer/deploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -x
+
+set -e
+
+FREETDS_VERSION=${FREETDS_VERSION:=1.1.6}
+
+./build.sh
```

Comment:
Should this now point to the Rakefile?

Reply:
With these tasks moved to a Rakefile, this file will be deleted.